etc/contracts-test-data/scripts/utils.ts

```diff
@@ -0,0 +1,159 @@
+import * as fs from 'fs';
```

Comment:
you are repeating the same file twice, that's not good, as it will be harder to manage and update in the future.

Could it be some shared library instead?